### PR TITLE
keypath: unify leader/primary path

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: Ref #8582 

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: pkg/utils/keypath/absolute_key_path.go [hunk 1]
```diff
@@ -57,6 +67,17 @@ func LeaderPath(p *MsParam) string {
 	return fmt.Sprintf(msLeaderPathFormat, ClusterID(), p.ServiceName)
 }
 
+// ExpectedPrimaryPath returns the expected_primary path.
+func ExpectedPrimaryPath(p *MsParam) string {
+	if p.ServiceName == "tso" {
```
[the remark this answers is not in the document]
Comment (okJiang) [reply pkg/utils/keypath/absolute_key_path.go]: done

**Review by rleungx — APPROVED** [review]

**Pull request merged**